designer/client/src/containers/Notifications.tsx

```diff
@@ -109,16 +109,14 @@ export function Notifications(): JSX.Element {
 
                 if (!isNetworkAccess) {
                     handleChangeConnectionError("NO_NETWORK_ACCESS");
-                } else if (possibleServerNotAvailableHttpStatuses.some((status) => status === error.response.status)) {
+                } else if (possibleServerNotAvailableHttpStatuses.some((status) => status === error?.response?.status)) {
                     handleChangeConnectionError("NO_BACKEND_ACCESS");
                 } else {
                     const errorResponseData = error?.response?.data || error.message;
-
+                    console.error(`Error: ${errorResponseData}`);
```

Comment:
Maybe it's better to move this console.error inside of the `NotificationActions.error` method to make it callable in all display error notifications? Also, I'm wondering shouldn't log more info like https status or response origin (it's good to know if the error origin was reverse proxy or backend server)